examples/path_mappings/foo.ts

```diff
@@ -1,7 +1,8 @@
-import num from '@myorg/js_lib'
 import { A } from '@myorg/dts_lib'
-
 export const a: A = 123
-export const b: string = `number: ${num}`
+console.log(a)
 
-console.log(a, b)
+// TODO: fix this before landing
+// import num from '@myorg/js_lib'
```

Comment:
Oh no, not a good sign that this breaking change is breaking enough to affect our examples 😣

Reply:
Its a very strange corner case example at least, https://github.com/aspect-build/rules_js/pull/675 will fix it

Reply:
```
js_library(
    name = "lib",
    # Provide index.js as declarations available to downstream type checking rules
    # since it has no corresponding .d.ts file.
    declarations = ["index.js"],
    visibility = ["//examples:__subpackages__"],
)
```